collect_app/src/main/java/org/odk/collect/android/preferences/screens/AdminPreferencesFragment.java

```diff
@@ -69,6 +74,22 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         findPreference("user_settings").setOnPreferenceClickListener(this);
         findPreference("form_entry").setOnPreferenceClickListener(this);
 
+        findPreference("project_name").setOnPreferenceChangeListener((preference, newValue) -> {
+            Project currentProject = currentProjectProvider.getCurrentProject();
+            projectsRepository.update(new Project(currentProject.getName(), currentProject.getIcon(), currentProject.getColor(), currentProject.getUuid()));
+            return true;
```

Comment:
If we do go with a `SummaryProvider` and `persistent=false` these listeners could return `false` to stop the value from getting saved to shared prefs.

Reply:
I have both but can't change that value to false because the summary is not updated.

Reply:
Hmmm I can have a play with it on Monday if you push what you have?